testsuite/cluster-test/src/cluster_swarm/cluster_swarm_kube.rs

```diff
@@ -248,7 +278,19 @@ impl ClusterSwarmKube {
         .await
     }
 
-    async fn run_jobs(&self, jobs: Vec<Job>, back_off_limit: u32) -> Result<()> {
+    pub async fn kill_job(&self, job_full_name: &str) -> Result<()> {
+        let dp = DeleteParams::default();
+        let job_api: Api<Job> = Api::namespaced(self.client.clone(), DEFAULT_NAMESPACE);
+        job_api
+            .delete(job_full_name, &dp)
+            .await?
+            .map_left(|o| println!("Deleting Job: {:?}", o.status))
+            .map_right(|s| println!("Deleted Job: {:?}", s));
+        Ok(())
```

Comment:
Do you want to also call into `wait_job_completion` here to wait until job is actually deleted?

Reply:
I figured it might make sense to leave that up to the user to decide, as per my note above. Jobs are already cleaned up during cluster-test runner teardown, so might make sense to batch all the deleting there? What do you think

Reply:
I think waiting is probably a safer bet.
I understand your argument about saving time, but I think most people aren't going to read to much into it, and will just assume that kill have job killed when succeeded. So waiting will probably provide a safer api this way.

Reply:
Got it, updated. Had to add a new param to `wait_job_completion` to handle a race if job doesn't exist by the time we start to wait for it.